climada_petals/hazard/wildfire.py

```diff
@@ -25,24 +25,42 @@
 import itertools
 from dataclasses import dataclass
 from datetime import date
+from pathlib import Path
+import copy
 
 import numpy as np
 import matplotlib.pyplot as plt
 import pandas as pd
 from scipy import sparse
+import scipy.ndimage # added by Sam G.
 from sklearn.neighbors import BallTree
 from sklearn.cluster import DBSCAN
 import numba
+import rasterio
+import shapely.geometry
+import random # added by Sam G.
 
 from climada.hazard.centroids.centr import Centroids
 from climada.hazard.base import Hazard
 from climada.hazard.tag import Tag as TagHazard
 from climada.util.constants import ONE_LAT_KM
 import climada.util.dates_times as u_dt
 import climada.util.coordinates as u_coord
+from climada.util.constants import (DEF_CRS) # Added by Sam G.
+
+import cartopy.crs as ccrs # added by Sam G.
+import cartopy.feature as cfeature # added by Sam G.
+from cartopy.mpl.gridliner import LONGITUDE_FORMATTER, LATITUDE_FORMATTER # added by Sam G.
+import rasterio.crs # added by Sam G.
+import rasterio.features # added by Sam G.
+import rasterio.mask # added by Sam G.
+import rasterio.warp # added by Sam G.
 
 LOGGER = logging.getLogger(__name__)
 
+BBOX = (-180, -85, 180, 85)  # [Lon min, lat min, lon max, lat max]
```

Comment:
Please choose a more descriptive name. Also, will this change in the future? Or is this for any ISIMIP data set? Then you could put it into the `util.constants`.